website/content/docs/operations/nomad-agent.mdx

```diff
@@ -82,22 +82,44 @@ There are several important messages that `nomad agent` outputs:
 
 ## Stopping an Agent
 
-An agent can be stopped in two ways: gracefully or forcefully. By default,
-any signal to an agent (interrupt, terminate, kill) will cause the agent
-to forcefully stop. Graceful termination can be configured by either
-setting `leave_on_interrupt` or `leave_on_terminate` to respond to the
+An agent can be stopped in two ways: gracefully or forcefully. By default, any
+stop signal to an agent (interrupt, terminate, kill) will cause the agent to
+forcefully stop. Graceful termination can be configured by either setting
+[`leave_on_interrupt`][] or [`leave_on_terminate`][] to respond to the
 respective signals.
 
-When gracefully exiting, clients will update their status to terminal on
-the servers so that tasks can be migrated to healthy agents. Servers
```

Comment:
The behavior described here for clients has never been the case. Only the node GC job ever calls `Node.Deregister` and the client calls `Node.UpdateStatus` only to heartbeat (i.e. status is only ever "ready"). 

Arguably if we have it documented here that makes the lack of it a bug which we should fix and backport. I'd definitely be open to that idea, seeing as how you have to opt-in to `leave_on_interrupt` or `leave_on_terminate` anyways so that's not going to break backwards compatibility unexpectedly.

Reply:
Sorry I missed the review, but I would be +1 on rewriting this paragraph altogether when your drain PRs land. I find these to be surprising and unhelpful definitions of "graceful" and "forceful." I suspect the only reason these wrong docs lived so long is that they're so wrong and unhelpful it wasn't clear whether they or Nomad needed fixing as you pointed out.

Reply:
Will do!